amanuensis/__init__.py

```diff
@@ -205,12 +206,17 @@ def _setup_arborist_client(app):
     if app.config.get("ARBORIST"):
         app.arborist = ArboristClient(arborist_base_url=config["ARBORIST"])
 
-def _setup_hubspot_key(app):
+def _setup_hubspot_client(app):
     if app.config.get("HUBSPOT"):
-        if "API_KEY" in config["HUBSPOT"]:
-            app.hubspot_api_key = config["HUBSPOT"]["API_KEY"]
-        # else:
-            #TODO throw error
+        try:
+            app.hubspot_access_token = config["HUBSPOT"]["ACCESS_TOKEN"]
+            app.hubspot_client = hubspot.Client.create(access_token=app.hubspot_access_token)
+        except KeyError as ex:
+            logger.exception(ex)
+            raise KeyError("Hubspot ACCESS_TOKEN not found")
```

Comment:
should we make adding Hubspot optional? Right now it can not start without and api key

Reply:
Yes, that's a good idea.